test/Push.hs

```diff
@@ -29,11 +34,28 @@ test = do
           case mkInterest txt of
             Nothing -> True
             Just _ -> False
-  describe "Notifications parse" $
-    it "Notification JSON parses correctly" $
-    property $
-    let (inputBS, expected) = validFCMDecoding
-    in A.decode inputBS == Just expected
+  describe "Notifications parse" $ do
+    it "A specific FCM Notification JSON parses correctly" $
```

Comment:
In general, can you make the test descriptions read more like English (taking into account the `it`)? The `describe` should explain what is under test, and the `it` should describe its behaviour.

e.g., this would read (and print) better as

```
describe "Notification JSON parser" $ do
    it "correctly parses a specific FCM Notification"
    ...
```